com.valvesoftware.Steam.yml

```diff
@@ -107,16 +107,17 @@ add-extensions:
   org.freedesktop.Platform.ffmpeg-full:
     directory: lib/ffmpeg
     add-ld-path: .
-    version: '21.08'
+    version: '22.08beta'
     no-autodownload: true
     autodelete: false
 
-  org.freedesktop.Platform.ffmpeg_full.i386:
-    directory: lib32/ffmpeg
-    add-ld-path: .
-    version: '21.08'
-    no-autodownload: true
-    autodelete: false
+#  TODO: freedesktop-sdk 22.08 doesn't currently have this
+#  org.freedesktop.Platform.ffmpeg_full.i386:
```

Comment:
We don't need to remove extension point for a non-existent extension. It'll just stay empty, and should be installed and enabled when/if it becomes available.

Reply:
Ah, cool.